### PR TITLE
 Remove Microsoft user names from source code

### PR DESCRIPTION
**Pull request by dannyvv**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/6125)

### CODE REVIEW

**Review by NickGerleman — APPROVED** [review]
As an FYI, we have a lot of these in our changelog files for folks who use their work email through Github. Those are also stored in commit history though.

I think we've been kind of bad with internal strings in general. Last I heard, opaque URLs like ADO issues are fine, and code names for stuff internal to Office leaking probably isn't an issue (we have  those still).

File: packages/E2ETest/windows/ReactUWPTestApp/Package.appxmanifest [hunk 1]
```diff
@@ -8,14 +8,14 @@
 
   <Identity
     Name="ReactUWPTestApp"
-    Publisher="CN=RNWTest"
+    Publisher="CN=react-native-windows-testing"
```
Comment (asklar) [inline packages/E2ETest/windows/ReactUWPTestApp/Package.appxmanifest:11]: does this rename work without regenerating the certificates?

**Review by vmoroz — APPROVED** [review]
:shipit:

File: packages/E2ETest/windows/ReactUWPTestApp/Package.appxmanifest (hunk 1 above)
Comment (dannyvv) [reply packages/E2ETest/windows/ReactUWPTestApp/Package.appxmanifest:11]: @asklar: `yarn windows` works fine for playground and e2etests.. 
Is there anything else you'd like me to verify?

**Pull request merged**